waspc/examples/todoApp/todoApp.wasp

```diff
@@ -34,7 +34,14 @@ app todoApp {
   },
   db: {
     system: PostgreSQL
-  }
+  },
+  emailSender: {
```

Comment:
[Head in the clouds]: what if we would make `emailSender` a top level declaration, and then you can have multiple of them, and when something is using email sender, you have to specify which one you want to use?
I wouldn't do this now, neither am I sure that we should ever do it, but I am mentioning this because I think it might be interesting wasp-design-wise -> this idea that some of this stuff could be standalone, and we could have multiple of them. Like `emailSender`, or `server`, or even `db`.
I guess with email sender, it could make some sense, the question is would anybody ever need it -> maybe they want to use different providers for different stuff hm? Weird. Also, does it make any sense to SendGrid providers? I don't know. But if it is standalone like that and named, then it can also be passed around as a dependency, maybe it can be imported from external module, ... .
Ok, just food for thought a bit, nothing you need to do on this PR.

Reply:
@infomiho bump on this one!

Reply:
I've read the comment, and didn't respond due to the note:

> Ok, just food for thought a bit, nothing you need to do on this PR.